internal/sdkv2provider/data_source_tunnel.go

```diff
@@ -59,7 +65,14 @@ func dataSourceCloudflareTunnelRead(ctx context.Context, d *schema.ResourceData,
 	accID := d.Get(consts.AccountIDSchemaKey).(string)
 
 	name := d.Get("name").(string)
-	tunnels, _, err := client.ListTunnels(ctx, cloudflare.AccountIdentifier(accID), cloudflare.TunnelListParams{Name: name})
+	params := cloudflare.TunnelListParams{Name: name}
+
+	if v, ok := d.GetOkExists("is_deleted"); ok {
+		isDeleted := v.(bool)
+		params.IsDeleted = &isDeleted
```

Comment:
```suggestion
		params.IsDeleted = cloudflare.BoolPtr(v.(bool))
```

Reply:
I was looking for exactly that and did not find it - thx.